src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,58 @@
+import { Link, NavLink } from 'react-router-dom';
+import { useFilters } from '../store/FilterContext';
+import classNames from 'classnames';
+
 export const PeopleFilters = () => {
+  const { query, sexs, centuries, setSearchWith, getSearchWith, searchParams } =
+    useFilters();
+
+  function handleQueryChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setSearchWith({ query: event.target.value });
+  }
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">
-          All
-        </a>
-        <a className="" href="#/people?sex=m">
-          Male
-        </a>
-        <a className="" href="#/people?sex=f">
-          Female
-        </a>
+        {['All', 'Male', 'Female'].map(sex => {
+          let normalizedSex = 'All';
+
+          switch (sex) {
+            case 'Male':
+              normalizedSex = 'm';
+              break;
+            case 'Female':
+              normalizedSex = 'f';
+          }
```

Comment:
The `switch` statement does not have a `default` case. It's a good practice to include a `default` case to handle unexpected values, even if it's just to return the current state or log an error.